packages/react/src/Hero/Hero.tsx

```diff
@@ -74,7 +74,9 @@ function HeroDescription({size = '400', weight, children}: PropsWithChildren<Her
 
 type HeroActions = {
   href: string
-} & Omit<ButtonBaseProps, 'variant'>
+  leadingVisual?: React.ReactNode
+  trailingVisual?: React.ReactNode
+} & Omit<ButtonBaseProps, 'trailingVisual' | 'leadingVisual'>
```

Comment:
This was one of the few ways I was able to make `leadingVisual` and `trailingVisual` play nice with Octicons with its `Icon` type. It seems that there's a mismatch in props against `ReactElement` that TypeScript was complaining about, which is only a problem when using `Button` here, and not directly. 

A alternative would be replacing `ReactElement` type in `Button` for `leadingVisual` and `trailingVisual` with `React.ReactNode`, but I only wanted to adjust the `Hero` component, and not change anything within `Button`.